<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <exportmacro>KST_EXPORT</exportmacro>
 <class>CSDDialogWidget</class>
 <widget class="QWidget" name="CSDDialogWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>549</width>
    <height>488</height>
   </rect>
  </property>
  <layout class="QGridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="2" column="0">
    <widget class="KstFFTOptions" name="_kstFFTOptions" native="true"/>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="buttonGroup2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Spectrogram Options</string>
     </property>
     <layout class="QGridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="textLabel1">
        <property name="text">
         <string>Window size:</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
        <property name="buddy">
         <cstring>_windowSize</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="_windowSize">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>999999999</number>
        </property>
        <property name="value">
         <number>500</number>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <spacer name="spacer4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>210</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="CurvePlacementWidget" name="_curvePlacement" native="true"/>
   </item>
   <item row="3" column="0">
    <widget class="QGroupBox" name="_imageOptionsGroup">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Image Options</string>
     </property>
     <layout class="QGridLayout">
      <item row="0" column="0">
       <widget class="ColorPaletteWidget" name="_colorPalette" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="GroupBox9_3">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Source</string>
     </property>
     <layout class="QGridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="TextLabel1_6_2_4">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Data vector:</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="VectorSelector" name="_vector" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ColorPaletteWidget</class>
   <extends>QWidget</extends>
   <header>colorpalettewidget.h</header>
  </customwidget>
  <customwidget>
   <class>KstFFTOptions</class>
   <extends>QWidget</extends>
   <header>fftoptionswidget.h</header>
  </customwidget>
  <customwidget>
   <class>CurvePlacementWidget</class>
   <extends>QWidget</extends>
   <header>curveplacementwidget.h</header>
  </customwidget>
  <customwidget>
   <class>VectorSelector</class>
   <extends>QWidget</extends>
   <header>vectorselector.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>_vector</tabstop>
  <tabstop>_windowSize</tabstop>
 </tabstops>
 <includes>
  <include location="global">kst_export.h</include>
  <include location="local">fftoptionswidget.h</include>
  <include location="local">curveplacementwidget.h</include>
  <include location="local">colorpalettewidget.h</include>
  <include location="local">vectorselector.h</include>
 </includes>
 <resources/>
 <connections/>
</ui>
